Product_Landing_Page/css/style.css

```diff
@@ -0,0 +1,327 @@
+    #page-logo > img {
+      align-items: center;
+      justify-content: center;
+
+    }
+}*/
+
+#emailform {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  height: 200px;
+  margin-top: 50px;
+}
+
+#emailform,h2 {
+  margin-bottom: 20px;
+
+}
+
+#emailform input[type="email"] {
+  max-width: 275px;
+  width: 100%;
+  padding: 5px;
+}
+
+#emailform input[type="submit"] , .btn{
+  max-width: 150px;
+  width: 100%;
+  height: 30px;
+  margin: 15px 0;
+  border: 0;
+  background-color: #f1c40f;
+}
+
+#emailform input[type="submit"]:hover , .btn:hover{
+  background-color: orange;
+  transition: background-color 1s;
+}
+
+@media (max-width: 650px) {
+  #emailform {
+    margin-top: 120px;
+  }
+}
+
+.features-section
+{
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.feature{
+  display: flex;
+  flex-direction: row;
+}
+
+.icons{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 125px;
+  width: 20vw;
+
+}
+
+.description{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 125px;
+  width: 80vw;
+  padding: 5px;
+}
+
+@media (max-width:600px) {
+  .icons{
+    display: none;
+  }
+  .description{
+    width: 100%;
+    text-align: center;
+    padding: 0;
+    height: 150px;
+  }
+}
+
+#video1{
+  display: none;
+}
+
+.video-section{
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+}
+
+.video-section > iframe{
+  max-width: 600px;
+  width : 100%;
+  margin-top:40px;
+
+}
+
+.pricing-section{
+
+  max-width: 1000px;
+  width: 100%;
+  margin-left:auto;
+  margin-right: auto;
+  margin-top: 40px;
+  margin-bottom: 0;
+  display: flex;
+  justify-content:space-around;
+  flex-direction: row;
+}
+
+.pricing-section > div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  width: 33.3%;
+  margin: 10px;
+  border: 1px solid #000;
+  border-radius: 3px;
+}
+
+.product > .price-heading {
+  background-color: #ddd;
+  color: black;
+  padding: 15px 0;
+  width: 100%;
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+.product > h2 {
+  margin-top: 15px;
+}
+
+.product > ol {
+  margin: 15px 0;
+}
+
+.product > ol > li {
+  padding: 5px 0;
+}
+
+.product > button {
+  border: 0;
+  margin: 15px 0;
+  background-color: #f1c40f;
+  font-weight: 400;
+}
+
+@media (max-width:600px) {
+  .pricing-section{
+    flex-direction: column;
+  }
+  .pricing-section > div{
+    max-width: 300px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 7px;
+    margin-bottom: 7px;
+
+  }
+}
+
+footer{
+  max-width: 1000px;
+  width: 100%;
+  margin-left:auto;
+  margin-right: auto;
+  margin-top: 40px;
+}
+
+footer .footer-navigational-bar{
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+footer > div > ul{
+  display: flex;
```

Comment:
This is very very bad way of styling. Please use classes and structure your styling accordingly.

Reply:
Done